pallets/loans-ref/src/lib.rs

```diff
@@ -846,32 +844,28 @@ pub mod pallet {
 		/// Set the maturity date of the loan to this instant.
 		pub fn expire(pool_id: PoolIdOf<T>, loan_id: T::LoanId) -> DispatchResult {
 			Self::update_active_loan(pool_id, loan_id, |loan| {
-				loan.set_maturity(T::Time::now());
+				loan.set_maturity(T::Time::now().as_secs());
```

Comment:
Thats a good one! TBH, we should probably create a wrapper around `trait Time`, like `trait TimeAsSecs` and use this bound everywhere. We had confusions with this multiple times now....